src/server/JasmineGraphInstanceService.cpp

```diff
@@ -640,7 +640,32 @@ void *instanceservicesession(void *dummyPt) {
 
 
             std::string aggregatedTriangles= JasmineGraphInstanceService::aggregateCentralStoreTriangles(graphId, partitionId, partitionIdList);
-            write(connFd, aggregatedTriangles.c_str(), aggregatedTriangles.size());
+
+            std::vector<std::string> chunksVector;
+
+            for (unsigned i = 0; i < aggregatedTriangles.length(); i += INSTANCE_DATA_LENGTH-10) {
+                std::string chunk = aggregatedTriangles.substr(i, INSTANCE_DATA_LENGTH-10);
+                if (i + INSTANCE_DATA_LENGTH-10 < aggregatedTriangles.length()) {
+                    chunk += "/SEND";
+                } else {
+                    chunk += "/CMPT";
+                }
+                chunksVector.push_back(chunk);
+            }
+
+            for (int loopCount=0;loopCount < chunksVector.size();loopCount++) {
```

Comment:
Add spaces between variables. Lets change as follows,

```
 for (int loopCount = 0; loopCount < chunksVector.size(); loopCount++) {
```

Reply:
Updated the code